web-ui/github.ml

```diff
@@ -337,24 +337,19 @@ let repo_handle ~meth ~owner ~name ~repo path =
       | Aborted | Failed _ | Passed | Undefined _ -> Lwt.return None
     in
     let cancel_many (commit: Client.Commit.t) (job_infos : Client.job_info list) =
-      let open Lwt.Infix in
-      Lwt_list.filter_map_p (fun job_info -> can_cancel commit job_info) job_infos >>= fun l ->
-        Lwt_list.map_p (fun (ji, j) -> Current_rpc.Job.cancel j |> Lwt_result.map @@ fun () -> ji) l >>= fun x ->
-          let success = ref [] in
-          let failed = ref 0 in
```

Comment:
Also note that using List.fold_left is usually a bit faster than references due to the GC handling references less efficiently.